supervisord.conf

```diff
@@ -0,0 +1,26 @@
+[supervisord]
+nodaemon=true
+
+[program:xl2tpd]
```

Comment:
I'm wondering whether we can make `run.sh` into a program here, and start `ipsec` and `xl2tpd` when the `run.sh` fires the `STOPPING` event?